build.cake

```diff
@@ -25,4 +28,4 @@ Task("Default")
     .IsDependentOn("Embed MediatR")
     .IsDependentOn("dotnetcore");
 
-RunTarget(Target);
```

Comment:
This is defined in the other package, and is picked up automatically.  The reason this stopped working correctly is due to the breaking changes to the bootstrapper scripts.